core/sail/fts/elasticsearch/src/main/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchIndex.java

```diff
@@ -173,7 +174,7 @@ public void initialize(Properties parameters)
 		throws Exception
 	{
 		super.initialize(parameters);
-		indexName = parameters.getProperty(INDEX_NAME_KEY, DEFAULT_INDEX_NAME);
+		indexName = parameters.getProperty(INDEX_NAME_KEY, UUID.randomUUID().toString());
```

Comment:
It should be set by the tests, but it doesn't have to be set and isn't required to be set, plus it will break code out there that uses the default value. I think the focus should be on making sure each test uses a different cluster name, not index.